cranelift/filetests/filetests/isa/riscv64/constants.clif

```diff
@@ -60,7 +60,7 @@ block0:
 }
 
 ; block0:
-;   auipc a0,0; ld a0,12(a0); j 12; .8byte 0xffff00000000
+;   auipc a0,.LConst; ld a0,.LConst(0xffff00000000)
```

Comment:
This label doesn't distinguish between two different constants.

Reply:
I don't really have a very good solution for this, we only know which label we get assigned after we emit the constant into the pool, and that only happens at emit time. When formatting the code we don't know which one its going to end up in.

Reply:
Oh, the actual immediate is put between parens. I read the parens as being part of the regular lw inst syntax, but in that case it would be more like `lw x2, 0xdeadbeef(x1)`, so my brain still parsed it wrong.

Reply:
It turns out that riscv has `li rd, imm` assembler mnemonic that is much mor readable. I've switched to using that.